sigs.yaml

```
@@ -2148,8 +2148,9 @@ sigs:
     owners:
     - https://raw.githubusercontent.com/kubernetes/node-problem-detector/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/test-infra/master/config/jobs/kubernetes/node-problem-detector/OWNERS
-  - name: noderesourcetopology-api
+  - name: resource-management
     owners:
+    - https://raw.githubusercontent.com/kubernetes/dynamic-resource-allocation/master/OWNERS
```

```suggestion
    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/staging/src/k8s.io/dynamic-resource-allocation/OWNERS
```

For staging repos, they should point to their location in k/k as that is the authoritative place for the code 👍

---

Updated thanks.